test/c_data_integration/src/utils.cpp

```diff
@@ -0,0 +1,143 @@
+// Copyright 2024 Man Group Operations Limited
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "sparrow/c_data_integration/utils.hpp"
+
+#include <charconv>
+#include <string_view>
+
+#include "sparrow/c_data_integration/constant.hpp"
+
+namespace sparrow::c_data_integration::utils
+{
+    std::vector<std::vector<std::byte>> hexStringsToBytes(const std::vector<std::string>& hexStrings)
```

Comment:
nitpicking: can we stick with snake case? This can be done in the next PR though.

Reply:
ok